src/CommunityToolkit.Mvvm/ComponentModel/Attributes/ObservablePropertyAttribute.cs

```diff
@@ -20,38 +20,54 @@ namespace CommunityToolkit.Mvvm.ComponentModel;
 /// partial class MyViewModel : ObservableObject
 /// {
 ///     [ObservableProperty]
-///     private string name;
+///     public partial string name { get; set; }
```

Comment:
Just a question: should "name" here also have a capital, since it's now a property? and the generated code below also seems to have a capital?